src/state-manager/TransactionQueue.ts

```diff
@@ -557,40 +642,16 @@ class TransactionQueue {
               /* prettier-ignore */ if (logFlags.error && logFlags.verbose) this.mainLogger.error(`broadcast_finalstate data == null`)
               continue
             }
+            const isValidFinalDataSender = this.factValidateCorrespondingTellFinalDataSender(queueEntry, header.sender_id)
```

Comment:
same above question here

Reply:
same as above.  I think there is no going back to  the old corresponding tell with POQo